orderbevestiging.php

```diff
@@ -28,6 +28,29 @@
 if (!isset($_SESSION["order"]["placeOrder"])) {
     $_SESSION["order"]["placeOrder"] = FALSE;
 }
+/**
+ * @param $content
+ * @param $naam
+ * @param $customerID
+ * @param $telefoonnummer
```

Comment:
Nice dat je een docstring gebruikt maar voegt het echt wat toe? En waar heb je dit vandaan als het chatgpt of iets is zal ik ook nog even nadenken of je dit erin wilt zetten.